docs/tutorials/excel-tutorial.md

```diff
@@ -20,7 +20,7 @@ This tutorial teaches you the basics of recording, editing, and writing an Offic
 
 First, you'll need some data and a small starting script.
 
-1. Create a new workbook in [Excel for the Web](https://www.office.com/launch/excel).
+1. Create a new workbook in [Excel for the Web](https://www.office.com/launch/excel). The Action Recorder is currently only available in Excel for the web, so that's where you must start this tutorial.
```

Comment:
Should the first instance of 'Excel for the web' have a lowercase 'w'?

Reply:
Probably...